sdk/agrifood/azure-agrifood-farming/CHANGELOG.md

```diff
@@ -1,6 +1,27 @@
 # Release History
 
 ## 1.0.0b2 (Unreleased)
```

Comment:
Either in this PR or sometime before release, you'll need to add your release date here